content/Software and Downloads/Installation/Add-a-library-to-Arduino-IDE.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Add a library to Arduino IDE"
+---
+
+Libraries are a collection of code that makes it easy to connect sensors, RTCs, Wi-Fi modules, RGB matrices and other components to your board.
```

Comment:
Libraries are not only used to enable connected components, they can be required to use for communication, data processing, etc.

https://www.arduino.cc/reference/en/libraries/

> The Arduino environment can be extended through the use of libraries, just like most programming platforms. Libraries provide extra functionality for use in sketches, e.g. working with hardware or manipulating data.

Phrases like "extending" or "extra functionality" are more general alternatives, of course following up with examples is fine.

Reply:
[e03644e](https://github.com/arduino/help-center-content/pull/82/commits/e03644eccf76981b46cc81aac7206f8357cbad20)